### DISCUSSION TITLE
Add SPUTNIK V

### DISCUSSION THREAD

**Discussion by shailesh** [pallupz/covid-vaccine-booking#160] [Ideas]
```
def get_vaccine_preference():
    print(
        "It seems you're trying to find a slot for your first dose. Do you have a vaccine preference?"
    )
    preference = input(
        "Enter 0 for No Preference, 1 for COVISHIELD, or 2 for COVAXIN, or 3 for SPUTNIK V. Default 0: "
    )
    preference = int(preference) if preference and int(preference) in [0, 1, 2, 3] else 0

    if preference == 1:
        return "COVISHIELD"
    elif preference == 2:
        return "COVAXIN"
    elif preference == 3:
        return "SPUTNIK V"
    else:
        return None
```

Got this from Portal runtime filter JS

`"COVISHIELD"==e&&this.appointmentForm.get("covishieldFilter").setValue(""),"COVAXIN"==e&&this.appointmentForm.get("covaxinFilter").setValue(""),"SPUTNIK V"==e&&this.appointmentForm.get("sputnikFilter").setValue("")`

entire JS file here: https://www.cowin.gov.in/4.59dc5b8fc585b9340380.js

![screenshot](https://user-images.githubusercontent.com/9050349/118401599-eaba9700-b683-11eb-9082-e3e99b8241e3.png)

<img width="1788" alt="Screenshot 2021-05-16 at 8 16 52 PM" src="https://user-images.githubusercontent.com/9050349/118401605-ee4e1e00-b683-11eb-8e2c-b2d035088534.png">

**Comment by pallupz** [comment]
Will do. Thanks!

**Comment by pallupz** [comment]
done

**Reply by shailesh** [reply]
Great 👍
